storage/innobase/include/api0api.h

```diff
@@ -651,22 +651,26 @@ ib_open_table_by_name(
 Insert a row to a table.
 @return	DB_SUCCESS or err code */
 
+#pragma namemanglingrule(fnparmtype, on)
```

Comment:
What is wrong with the prototype? Is the function implementing this written with different types? Note that some external program could use InnoDB internals using this function and then you should not change types or name mangling rules.